src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,19 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int specialBulldozerField;
+
     public Bulldozer() {
     }
 
```

Comment:
it will be good decision to add one more constructor and use super method for the abstract class constructor(you can add it )